iml-postgres/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["IML Team <iml@whamcloud.com>"]
 edition = "2018"
 
 [dependencies]
-tokio-postgres = "0.4.0-rc.2"
+tokio-postgres = "0.4.0-rc.3"
```

Comment:
Just making sure we're comfortable using RC 3 instead of 0.3.0?

Reply:
Yes, haven't had any issues.